### ISSUE TITLE
Typo `//` in filepath prevents installation?

### ISSUE THREAD

**Issue by cgpu** [kazutan/tubeplayR#5]
@kazutan Really cool package!

Was not able to install it but when I do `devtools::source_url("..../.../tubeplay.R")` the function works so no problem there.

You might want to check the filepath for a possible typo, there are two consecutive slashes __`0gp/T//Rtm`__, check below installation attempt output and screenshot.

``` ‘/var/folders/82/qyksk10j3r397fs8566sfy580000gp/T//RtmppLvWPb/file17dc050b53ec0/tubeplayR_0.1.3.tar.gz’
```

![image](https://user-images.githubusercontent.com/38183826/56868098-dac0b380-69e5-11e9-916f-cd7db4d8def8.png)

```
> devtools::install_github("kazutan/tubeplayR")
Downloading GitHub repo kazutan/tubeplayR@master
✔  checking for file ‘/private/var/folders/82/qyksk10j3r397fs8566sfy580000gp/T/RtmppLvWPb/remotes17dc0286dec57/kazutan-tubeplayR-0d8a228/DESCRIPTION’ ...
─  preparing ‘tubeplayR’:
✔  checking DESCRIPTION meta-information ...
─  cleaning src
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘tubeplayR_0.1.3.tar.gz’
   
* installing *source* package ‘tubeplayR’ ...
** libs
Error: (converted from warning) no source files found
* removing ‘/Library/Frameworks/R.framework/Versions/3.5/Resources/library/tubeplayR’
Error in i.p(...) : 
  (converted from warning) installation of package ‘/var/folders/82/qyksk10j3r397fs8566sfy580000gp/T//RtmppLvWPb/file17dc050b53ec0/tubeplayR_0.1.3.tar.gz’ had non-zero exit status
> 
> devtools::source_url("https://raw.githubusercontent.com/cgpu/tubeplayR/master/R/tubeplay.R")
SHA-1 hash of file is 46e7d496f3849243f12d49c3a9bcd5e9d5533337
> tubeplay(url = "https://www.youtube.com/watch?v=oVQqmH4ufCQ")
```

**Comment by kazutan** [comment]
@cgpu 
I was sorry for the late reply.
I will confirm, so please wait a little more.

**Comment by kazutan** [comment]
I'm sorry that it took a long time.
I thought this issue was identical to #6 .
This issue will be closed as this has been fixed in tubeplayR v0.1.4.
